doc/users-guide/users-guide-pktio.adoc

```diff
@@ -500,97 +121,7 @@ of type *odp_pktin_queue_t* and are retrieved by threads calling the
- * and may invalidate all previous queue handles.
- *
- * @param pktio    Packet IO handle
- * @param param    Packet input queue configuration parameters. Uses defaults
- *                 when NULL.
- *
- * @retval 0 on success
- * @retval <0 on failure
- *
- * @see odp_pktio_capability(), odp_pktin_queue(), odp_pktin_event_queue()
- */
-int odp_pktin_queue_config(odp_pktio_t pktio,
-			   const odp_pktin_queue_param_t *param);
------
-The second argument to this call is the *odp_pktin_queue_param_t*
-[source,c]
------
-/**
- * Packet input queue parameters
- */
-typedef struct odp_pktin_queue_param_t {
-	/** Operation mode
-	  *
-	  * The default value is ODP_PKTIO_OP_MT. Application may enable
-	  * performance optimization by defining ODP_PKTIO_OP_MT_UNSAFE when
-	  * applicable. */
-	odp_pktio_op_mode_t op_mode;
-
-	/** Enable classifier
-	  *
-	  * * 0: Classifier is disabled (default)
-	  * * 1: Classifier is enabled. Use classifier to direct incoming
-	  *      packets into pktin event queues. Classifier can be enabled
-	  *      only in ODP_PKTIN_MODE_SCHED and ODP_PKTIN_MODE_QUEUE modes.
-	  *      Both classifier and hashing cannot be enabled simultaneously
-	  *      ('hash_enable' must be 0). */
-	odp_bool_t classifier_enable;
-
-	/** Enable flow hashing
-	  *
-	  * * 0: Do not hash flows (default)
-	  * * 1: Enable flow hashing. Use flow hashing to spread incoming
-	  *      packets into input queues. Hashing can be enabled in all
-	  *      modes. Both classifier and hashing cannot be enabled
-	  *      simultaneously ('classifier_enable' must be 0). */
-	odp_bool_t hash_enable;
-
-	/** Protocol field selection for hashing
-	  *
-	  * Multiple protocols can be selected. Ignored when 'hash_enable' is
-	  * zero. The default value is all bits zero. */
-	odp_pktin_hash_proto_t hash_proto;
-
-	/** Number of input queues to be created
-	  *
-	  * When classifier is enabled in odp_pktin_queue_config() this
-	  * value is ignored, otherwise at least one queue is required.
-	  * More than one input queues require flow hashing configured.
-	  * The maximum value is defined by pktio capability 'max_input_queues'.
-	  * Queue type is defined by the input mode. The default value is 1. */
-	unsigned num_queues;
-
-	/** Queue parameters
-	  *
-	  * These are used for input queue creation in ODP_PKTIN_MODE_QUEUE
-	  * or ODP_PKTIN_MODE_SCHED modes. Scheduler parameters are considered
-	  * only in ODP_PKTIN_MODE_SCHED mode. Default values are defined in
-	  * odp_queue_param_t documentation.
-	  * When classifier is enabled in odp_pktin_queue_config() this
-	  * value is ignored. */
-	odp_queue_param_t queue_param;
-
-} odp_pktin_queue_param_t;
------
-Note that the *queue_param* field of this struct is ignored in DIRECT mode.
-The purpose of `odp_pktin_queue_config()` is to specify the number of PktIn
+`odp_pktin_queue_config()` API, which purpose is to specify the number of PktIn
```

Comment:
The 'which' does not sound quite right to me. I would probably use 'whose', or if that is not good, I would say "... API, the purpose of which is ...".